src/main/java/frc/trigon/robot/subsystems/MotorSubsystem.java

```diff
@@ -40,6 +43,16 @@ public MotorSubsystem() {
         REGISTERED_SUBSYSTEMS.add(this);
     }
 
+    @Override
+    public void periodic() {
+        if (RobotHardwareStats.isReplay())
+            updateMechanisms();
+        else if (SHOULD_UPDATE_MECHANISMS.get())
+            updateMechanisms();
+        updatePeriodically();
+        super.periodic();
+    }
```

Comment:
* No need for `super.periodic()`
* `updatePeriodically()` should be called before the mechanisms